services/servicenow/service.go

```diff
@@ -73,18 +73,25 @@ func (s *Service) StateChangesOnly() bool {
 }
 
 type testOptions struct {
-	AlertID string      `json:"alert_id"`
-	Source  string      `json:"source"`
-	Level   alert.Level `json:"level"`
-	Message string      `json:"message"`
+	AlertID        string                 `json:"alert_id"`
+	Message        string                 `json:"message"`
+	Level          alert.Level            `json:"level"`
+	Source         string                 `json:"source"`
+	Node           string                 `json:"node"`
```

Comment:
These new properties are not used in the Test function yet.

Reply:
I hope it's fixed now.